lib/components/Editor/index.js

```diff
@@ -136,6 +138,14 @@ const Tiptap = (
       ) : (
         <EditorContent editor={editor} {...otherProps} />
       )}
+      <p className="neeto-editor-character-count">
+        {characterLimit
+          ? `${
+              characterLimit - editor?.storage?.characterCount?.characters()
+            } characters remaining`
```

Comment:
Can we extract this `p` tag and content to separate component which may take `characterLimit` and `editor` as props. The index file is getting to be too lengthy and messy